addons/io/org.openhab.io.imperihome/README.md

```diff
@@ -0,0 +1,355 @@
+# ImperiHome integration service
+
+This IO service exposes OpenHAB Items to the Evertygo [ImperiHome](http://www.evertygo.com/imperihome) dashboard app for Android and iOS.
+It creates a REST service at ```/imperihome/iss``` that implements the [ImperiHome Standard System API](http://dev.evertygo.com/api/iss) (ISS).
+
+## Installation
+
+At the moment the ImperiHome integration service is not part of the OpenHAB distribution and can't be installed using the Paper UI.
+If you want to try it out, [download the latest build](https://dl.dvelop.nl/oh-imperihome/) and drop the JAR file in the ```addons``` directory of OpenHAB.
+
+## Configuration
+
+The service itself has no configuration. ImperiHome on the other hand must be configured to connect to your OpenHAB instance.
+
+Start ImperiHome, open the menu and go to My Systems. Add a new system (+) and choose 'ImperiHome Standard System' as the object type. Now enter the URL to your OpenHAB instance
+ as Local URL, followed by ```/imperihome/iss```. For example, if your OH instance is running at ```http://192.168.1.10:8080/```, the Local URL would be ```http://192.168.1.10:8080/imperihome/iss```. 
+
+If you have port forwarding or similar set up to access your OH form the internet, you can also fill the Remote URL in the same way. For example: 
+```http://my-openhab-url.dyndns.org:8080/imperihome/iss```. Please be aware that this service provides no authentication mechanism, so anyone could use the API to control your 
```

Comment:
Please add empty lines before and after all code sections, since otherwise the layout will be broken through Jekyll on docs.openhab.org.